### Task
1. It is assumed that the mean systolic blood pressure is `μ = 120 mm Hg`. In the Honolulu Heart Study, a sample of `n = 100` people had an average systolic blood pressure of 130.1 mm Hg with a standard deviation of 21.21 mm Hg. Is the group significantly different (with respect to systolic blood pressure!) from the regular population?

   - Set up the hypothesis test.
   - Write down all the steps followed for setting up the test.
   - Calculate the test statistic by hand and also code it in Python. It should be 4.76190. We will take a look at how to make decisions based on this calculated value.



#### Answer:

Nullhypothesis:
People with certain heart disease (like the ones regarding the Honolulu Heart Study) have the same mean  systolic blood pressure of 
120 mm Hg like it is the mean from the whole population. 

Alternative Hypothesis:
Peolpe with those kind of heart disease have an higher average systolic pressure than the mean of 120 mm Hg.

It is a one tailed t-test. Because the alternative Hypothesis states, that the diseased people mean would be higher than the whole population mean.

degree of freedom df= 100-1 = 99

For the one tailed t-test and df = 99: t.95=1.660

To fail the Null Hypothesis with a p-value of 5% the statistic value must be higher than t.95.

In [1]:
import math

sample_mean = 130.1
pop_mean = 120
pop_std = 21.21
n = 100

statistic = (sample_mean - pop_mean)/(pop_std/math.sqrt(n))
statistic

4.761904761904759

the statistic value of 4.76 being > t.95=1.660 rejects the Null Hypothesis with a p-value of 5%.
So it would be very suprisingly, if the diseased patients would have the same low mean systolic
blood pressure.
